osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -227,6 +230,8 @@ public enum OsuSetting
         IntroSequence,
         UIHoldActivationDelay,
         HitLighting,
-        MenuBackgroundSource
+        MenuBackgroundSource,
+        ShowPlayfieldArea,
+        PlayfieldAreaDimLevel
```

Comment:
I'm not convinced this setting should exist. At least not until we add extended support for UI skinning which is a long way away. It's arguably out of scope of the linked issue anyway.

The name is misleading, too. It controls dim of the *border* and not the actual area inside.